lib/application/pages/home/collections/update/update_collection_memos.dart

```diff
@@ -1,8 +1,107 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:layoutr/common_layout.dart';
+import 'package:memo/application/constants/dimensions.dart' as dimens;
+import 'package:memo/application/constants/images.dart' as images;
+import 'package:memo/application/constants/strings.dart' as strings;
+import 'package:memo/application/theme/theme_controller.dart';
 
 class UpdateCollectionMemos extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Container(color: Colors.blue, child: Center(child: Text('Memos')));
+    return _CreateMemoEmptyState(
+      onTap: () {
+        print("teste");
+      },
+    );
   }
 }
+
+/// An empty state CTA to add a new `Memo` to a `Collection`.
+class _CreateMemoEmptyState extends ConsumerWidget {
+  const _CreateMemoEmptyState({this.onTap});
+
+  final VoidCallback? onTap;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final theme = ref.watch(themeController);
+    final textTheme = Theme.of(context).textTheme;
+
+    return Material(
```

Comment:
Maybe we could create one or two smaller widgets (could definitely be variables), but it would make this widget more readable.